CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## 2.8.0 (TDB)
+
+- Feature: Include the optional Ambassador Agent
```

Comment:
please add some comments about what it does. This will be the body of the release notes